scripts/generate_driver_wrappers.py

```diff
@@ -176,24 +176,31 @@ def main() -> int:
                         help='output file\'s location')
     args = parser.parse_args()
 
-    mbedtls_root = os.path.abspath(args.mbedtls_root)
+    repo_root = os.path.abspath(args.repo_root)
+
+    library_dir = ''
```

Comment:
Useless line: in Python, you can initialize a variable in each branch of an if/else.